keymaps/latex.json

```diff
@@ -1,7 +1,11 @@
 {
+  "atom-text-editor": {
+    "ctrl-alt-l": "latex:toggle-log"
+  },
   "atom-text-editor[data-grammar~=\"latex\"]": {
     "ctrl-alt-b": "latex:build",
     "ctrl-alt-s": "latex:sync",
-    "ctrl-alt-c": "latex:clean"
+    "ctrl-alt-c": "latex:clean",
+    "ctrl-alt-e": "latex:sync-log"
```

Comment:
Ok. Sounds good. Maybe one for `latex:sync` also since we know some user's do not know about that one either?